studies/models.py

```diff
@@ -133,6 +181,35 @@ class Study(models.Model):
     compensation_description = models.TextField(blank=True)
     criteria_expression = models.TextField(blank=True)
 
+    # Groups
+    # The related_name convention seems silly, but django complains about reverse
+    # accessor clashes if these aren't unique :/ regardless, we won't be using
+    # the reverse accessors much so it doesn't really matter.
+    preview_group = models.OneToOneField(
+        Group, related_name="study_to_preview", on_delete=models.SET_NULL, null=True
+    )
+    design_group = models.OneToOneField(
+        Group, related_name="study_to_design", on_delete=models.SET_NULL, null=True
+    )
+    analysis_group = models.OneToOneField(
+        Group, related_name="study_for_analysis", on_delete=models.SET_NULL, null=True
+    )
+    submission_processor_group = models.OneToOneField(
+        Group,
+        related_name="study_for_submission_processing",
+        on_delete=models.SET_NULL,
+        null=True,
+    )
+    researcher_group = models.OneToOneField(
+        Group, related_name="study_for_research", on_delete=models.SET_NULL, null=True
+    )
+    manager_group = models.OneToOneField(
+        Group, related_name="study_to_manage", on_delete=models.SET_NULL, null=True
+    )
+    admin_group = models.OneToOneField(
+        Group, related_name="study_to_administer", on_delete=models.SET_NULL, null=True
+    )
```

Comment:
Is there a reason that some of the `related_name`s are `study_to_...` and some are `study_for_...`?